operator/docs/prologue/quickstart.md

```diff
@@ -16,7 +16,7 @@ One page summary on how to start with Loki Operator and LokiStack.
 
 ## Requirements
 
-The easiest way to start with the Loki Operator is to use Kubernetes [kind](sigs.k8s.io/kind).
+The easiest way to start with the Loki Operator is to use Kubernetes [kind](https://github.com/kubernetes-sigs/kind).
```

Comment:
Thanks for noticing this issue :+1: 

I think we meant to link to the "website" instead of the source though: https://kind.sigs.k8s.io/
I think this might be more useful for someone looking into what kind is. Do you agree?

Reply:
I think this is a better proposal, which will allow developers to understand the content better.